### PR TITLE
[SDK-core] Compile AJV validations

### PR DESCRIPTION
**Pull request by kasparkallas**
Why?
* AJV without compilation uses `eval` internally which is not OK in some environments (like Google Chrome extension)
* BONUS: smaller bundle size

Backstory?
Got feedback on Discord that a user couldn't use SDK-core inside a Google Chrome extension.

Links to Discord discussion:
* https://discord.com/channels/752490247643725875/752493348169711696/992939991141453875
* https://discord.com/channels/752490247643725875/752493348169711696/993421190318280734
* https://discord.com/channels/752490247643725875/752493348169711696/994044873516781578

### CODE REVIEW

**Comment by kasparkallas** [comment]
It was reported on Discord that this indeed fixed the issue with the Chrome Extension the user was having.

**Comment by 0xdavinchee** [comment]
@kasparkallas is this ready for review?

File: packages/sdk-core/scripts/ajv.ts [hunk 1]
```diff
@@ -90,39 +85,26 @@ const streamRequestSchema: JSONSchemaType<IStreamRequestFilter> = {
...
+    code: {
+        lines: true, // Please format after generation.
+        optimize: true,
+        source: true,
+        esm: true,
+        formats: _`require("./ajvCustomFormats.js")`,
+    },
+});
+
+let moduleCode = standaloneCode(ajv);
```
Comment (0xdavinchee) [inline packages/sdk-core/scripts/ajv.ts:107]: just out of curiosity, where did you get the idea to do it like this

Comment (kasparkallas) [reply packages/sdk-core/scripts/ajv.ts:107]: The community member recommended it who was having the `unsafe-eval` issue with the Chrome Extension.

**Review by 0xdavinchee — APPROVED** [review]
Looks good - if tests pass, LGTM

File: packages/sdk-core/scripts/ajv.ts (hunk 1 above)
Comment (0xdavinchee) [reply packages/sdk-core/scripts/ajv.ts:107]: gotcha

**Pull request merged**